src/main/js/wiki/components/FileBrowser.tsx

```diff
@@ -41,7 +41,7 @@ class FileBrowser extends React.Component<Props> {
     const children = orderFiles(this.props.directory.children);
 
     return (
-      <ul className="list-unstyled">
+      <ul className="list-unstyled" style={{ borderTop: "1px solid #ddd" }}>
```

Comment:
use the style object for a consistent usage

Reply:
I keep struggling to find the proper syntax for this refactoring :disappointed: Help plz?

Reply:
Thanks :D